packages/account/src/Sections/Verification/ProofOfOwnership/FileUploader.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { Button, Input, Icon } from '@deriv/components';
+import classNames from 'classnames';
+
+const FileUploader = ({
+    handleFile,
+    file_name,
+    class_name,
+    data_test_id,
+    name,
+    error,
+    validateField,
+    index,
+    sub_index,
+    updateErrors,
+    show_button,
+    updateShowButton,
+}) => {
+    // Create a reference to the hidden file input element
+    const hidden_file_input = React.useRef(null);
+    const handleClick = e => {
+        e.stopPropagation();
+        e.nativeEvent.stopImmediatePropagation();
+        hidden_file_input.current.click();
+    };
+
+    const handleChange = event => {
+        event.stopPropagation();
+        event.nativeEvent.stopImmediatePropagation();
+        const fileUploaded = event.target.files[0];
+        handleFile(name, fileUploaded);
+        updateShowButton(sub_index, !show_button);
+    };
+    const handleIconClick = e => {
+        e.stopPropagation();
+        e.nativeEvent.stopImmediatePropagation();
+        updateShowButton(sub_index, !show_button);
+        e.target.parentElement.parentElement.parentElement.querySelector(`input[name="${name}"]`).value = '';
```

Comment:
these long chains are very prone for errors. i think we need to deeply use optional chaining in these areas

Reply:
I will change this.

Thank you